### PR TITLE
Docs fixes

### PR DESCRIPTION
**Pull request by xBalbinus**
Closes: [#92](https://github.com/osmosis-labs/docs/issues/92)

## What is the purpose of the change

Parts of the modules docs point to invalid pages caused by invalid links in the new docbuild process. Fixing these in this PR.

## Brief Changelog

Combined the separate md's in spec folders into one big README's.

## Testing and Verifying

This change added tests and can be verified as follows:

Copy and paste the MD onto the docs repo to test.

## Documentation and Release Note

  - Does this pull request introduce a new feature or user-facing behavior changes? (yes / **no**)
  - Is a relevant changelog entry added to the `Unreleased` section in `CHANGELOG.md`? (yes / **no**)
  - How is the feature or change documented? (not applicable   /   specification (`x/<module>/spec/`)  /  **[Osmosis docs repo]**(https://github.com/osmosis-labs/docs)   /   not documented)

### CODE REVIEW

**Comment by xBalbinus** [comment]
Note, tables in lockup specs need fixing.

**Comment by xBalbinus** [comment]
All merge conflicts resolved! Closes [#92](https://github.com/osmosis-labs/docs/issues/92) part 2.

File: .markdownlint.yml [hunk 1]
```diff
@@ -1,5 +1,9 @@
+{
```
Comment (alexanderbez) [inline .markdownlint.yml:1]: errr, the markdownlint config is described in YAML, hence the file extension name. I'm pretty confident JSON is supported to, but why make this change?

Comment (xBalbinus) [reply .markdownlint.yml:1]: Ah, I'll put it back into YAML, I was just making changes based on some docs that described how to disable certain linting tests and those docs happened to be in JSON

**Pull request merged**